Podfile

```diff
@@ -1,8 +1,7 @@
-platform :osx, '10.9'
+platform :osx, '10.12'
```

Comment:
Why this bump? Are we now using APIs in SwiftGenKit that are only available in 10.12?